### ISSUE TITLE
Is it possible to change the justify without changing the {/an} alignment?

### ISSUE THREAD

**Issue by ilman01** [arcusmaximus/YTSubConverter#78]
Is it possible to change the justify without changing the {/an} alignment?

If you write {/an1}, it will align to the left and also justify  to the left. 

What I want to do is have the alignment as {/an2} but have it justify to the left instead of the center, without having to use {/pos} in combination with {/an1}.

Example video: https://youtu.be/sdxARQoF_C4 (you have to view it on PC) 

The example video I gave did not use {/pos}, because on mobile it is still just a normal {/an2}, but on PC it is {/an2} with justify left.

If this feature isn't available, I would love to have it as an ASS override tag that can be converted into .ytt

**Comment by ilman01** [comment]
Update: I managed to recreate the example video's subtitle using premiere pro COA-608 format subtitle and exporting it as .xml

Premiere pro has the option to change the justification seperately from the alignment.

It does have some quirks. If the longest line is has more than 20 characters, it will align just fine. But if the longest line has 20 characters or less, it will move to the left, to fix this I set the "Set Horizontal Position" to 144 and that somehow works to align it to the middle again on youtube.

I'll attach the subtitle to this comment.
[Dove Cameron - My Once Upon a Time (From 'Descendants 3') [sdxARQoF_C4].zip](https://github.com/arcusmaximus/YTSubConverter/files/9689620/Dove.Cameron.-.My.Once.Upon.a.Time.From.Descendants.3.sdxARQoF_C4.zip)

I don't know if this is possible with .ytt but maybe someone can figure out if it's possible?
